### PR TITLE
Simple Forms: Pull pdf upload out into a service

### PR DESCRIPTION
**Pull request by Thrillberg**
## Summary
This refactor PR is in anticipation of needing to upload PDFs directly from the front end for the upcoming **Form Upload** functionality. For that functionality we'll want to bypass the PDF filling and stamping and submit directly to the Benefits Intake API.

## Related issue(s)
https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/1257

## Testing done
New test written and passing.

### CODE REVIEW

File: modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
 require 'ddtrace'
-require 'simple_forms_api_submission/service'
 require 'simple_forms_api_submission/metadata_validator'
-require 'simple_forms_api_submission/s3'
```
Comment (Thrillberg) [inline modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb:6]: This deletion is unrelated and should have been done earlier when I removed CHAMPVA code from this controller.

**Review by pennja — APPROVED** [review]
Looks good, great work! 👍
